evalml/data_checks/id_columns_data_check.py

```diff
@@ -180,16 +181,19 @@ def validate(self, X, y=None):
         ]  # columns whose name is "id"
         id_cols = {col: 0.95 for col in cols_named_id}
 
-        for dtypes in [["Double"], ["Integer", "IntegerNullable", "Categorical"]]:
-            X_temp = X.ww.select(include=dtypes)
+        for types in [
+            ["Double"],
+            ["Integer", "IntegerNullable", "Categorical", "Text", "Unknown"],
```

Comment:
Integrated with the UI, Text columns are `Text` type, whereas within evalml, Text columns are `Unknown` type. I'm not sure how text columns become `Text` type in the UI when there is no `Text` logical type in Woodwork

Reply:
@jeff-hernandez any chance you'd know why it's coming in as `Text`?